### ISSUE TITLE
Hyperparameter to reproduce Table 2

### ISSUE THREAD

**Issue by chenwydj** [clovaai/wsolevaluation#22]
Could you also provide the recommended command/hyperparameter settings to reproduce Table 2?

Especially for the learning rate/batch size/epochs etc. for different methods with different backbones on three datasets.

This would benefit to a fair comparison who will cite your work.

Thank you very much!!

**Comment by junsukchoe** [comment]
Thanks for your interest in our work!

We will release the hyperparameters for Table 2 soon. However, we strongly recommend you to refer our latest benchmark with new metric (`MaxBoxAccV2`). Corresponding hyperparameters can be found at [here](https://docs.google.com/spreadsheets/d/1O4gu69FOOooPoTTtAEmFdfjs2K0EtFneYWQFk8rNqzw/edit#gid=0).

**Comment by coallaoh** [comment]
Closing the issue, assuming the question was answered :)
Please re-open the issue as necessary.
